test/parallel/test-child-process-exec-maxBuffer.js

```diff
@@ -10,6 +10,29 @@ function runChecks(err, stdio, streamName, expected) {
   assert.deepStrictEqual(stdio[streamName], expected);
 }
 
+// default value
+{
+  const cmd = `"${process.execPath}" -e "console.log('a'.repeat(200 * 1024))"`;
+
+  cp.exec(cmd, common.mustCall((err) => {
+    assert.strictEqual(err.message, 'stdout maxBuffer length exceeded');
+    assert(err instanceof RangeError);
```

Comment:
Nit: it would be better to check this first.

Reply:
I fixed it!